winml/lib/Api.Image/EventTimer.h

```diff
@@ -0,0 +1,29 @@
+#include "pch.h"
+
+class EventTimer
+{
+public:
+  EventTimer()
+  {
+  }
+  
+  bool Start()
+  {
+    auto now = std::chrono::high_resolution_clock::now();
```

Comment:
Would it be better to let "using CLOCK=..." to shorten the symbol?

Reply:
I like being explicit when it's in a header to be certain to not accidentally pollute other files that include the header.